web_generator/lib/src/config.dart

```diff
@@ -154,10 +158,17 @@ class YamlConfig implements Config {
       throw TypeError();
     }
 
+    final allFiles = fs.globSync(
```

Comment:
nit: The `globSync` code seems to be fairly similar everytime `globSync` is used, maybe move it to a helper function?